transports/transports.go

```diff
@@ -69,3 +69,16 @@ func Register(t types.ImageTransport) {
 func ImageName(ref types.ImageReference) string {
 	return ref.Transport().Name() + ":" + ref.StringWithinTransport()
 }
+
+// List returns a comma separate list of transports
+func ListNames() string {
```

Comment:
* While every other access is protected by the `kt.mu` mutex, this should probably be as well
* Return an array of strings, and let the caller join with `strings.Join(theList, ", ")`? That seems a bit cleaner than returning a single string and possibly forcing callers to split it and recombine.